src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1698,12 +1707,12 @@ msgstr "Nachrichten"
 
 #: templates/_base.html:194
 msgid "All push notifications"
```

Comment:
Please also adapt the english source strings in the base template here:
https://github.com/Integreat/integreat-cms/blob/ef44282cb1cd16ddaa5fab7ceeefca47629adca3/src/cms/templates/_base.html#L194
"All push notifications" => "All news messages"
https://github.com/Integreat/integreat-cms/blob/ef44282cb1cd16ddaa5fab7ceeefca47629adca3/src/cms/templates/_base.html#L198
"Create push notification" => "Create news message"